.cargo/config.toml

```diff
@@ -1,6 +1,6 @@
 [target.'cfg(all(target_arch = "arm", target_os = "none"))']
 # replace STM32F103C8 with your chip as listed in `probe-rs chip list`
-runner = "probe-rs run --chip STM32F103C8"
+runner = "probe-rs run --chip STM32F103CBT6"
```

Comment:
Is there a reason to change the stm32f103C8 --> stm32f103CBT6?
The 64k --> 128k difference means we would have compatibility issues with normal bluepill boards/controllers if the binary gets too big.

Maybe checking what the release binary size is compared to the debug release can tell us more about what's eating all the space if the size is the issue.

Reply:
I was just reading the writing on the chips we have. All of mine end in CBT6, might as well have the extra space available if it exists